sos/plugins/openstack_cinder.py

```diff
@@ -45,6 +47,9 @@ def setup(self):
         else:
             self.add_copy_spec_limit("/var/log/cinder/*.log",
                                      sizelimit=self.limit)
+        if self.get_option("exclude"):
+            exclude = self.get_option("exclude")
+            self.add_forbidden_path(exclude)
```

Comment:
Heh! Well spotted - those are there to make the `opts` format be both readable and pass pep8 at the same time... :)